heartbeat/monitors/active/http/http_test.go

```diff
@@ -808,6 +810,57 @@ func mustParseURL(t *testing.T, url string) *url.URL {
 	return parsed
 }
 
+// helper that compresses some content as gzip
+func gzipBuffer(t *testing.T, toZip string) *bytes.Buffer {
+	var gzipBuffer bytes.Buffer
+	gzipWriter := gzip.NewWriter(&gzipBuffer)
+	defer gzipWriter.Close()
+	_, err := gzipWriter.Write([]byte(toZip))
+	if err != nil {
+		t.Fatal(err)
+	}
+	return &gzipBuffer
+}
+
+func TestDecodesGzip(t *testing.T) {
+	gzip := gzipBuffer(t, "TestEncodingAccept")
```

Comment:
Let's not shadow the `gzip` package with a new local variable

Reply:
https://github.com/elastic/beats/pull/28739/commits/32c51a628472cb1bd11de9a7d3c1e10f3259308e